node/Cargo.toml

```diff
@@ -7,4 +7,15 @@ license = "MIT"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# Local dependencies
+blockchain-storage = { path = "../blockchain_storage/" }
+common = { path = "../common/" }
+consensus = { path = "../consensus/" }
 logging = { path = "../logging/" }
+rpc = { path = "../rpc/" }
+subsystem = { path = "../subsystem/" }
+
+# External dependencies
+anyhow = "1.0"
+clap = { version = "3.1", features = [ "derive" ] }
+tokio = "1.17"
```

Comment:
do you need everything from tokio?

Reply:
Good point, will check. Probably could be narrowed down somewhat.

Reply:
It seems that by default, tokio does not pull in any functionality other than bare minimum. Added `default-features = false` to the manifest anyway just to be sure